### PR TITLE
Use ugettext for all NotFound exception messages

### PR DESCRIPTION
**Pull request by mutantsan**
There are a many strings in CKAN that aren't wrapped with **ugettext**. I've started from NotFound exception (I don't know why:)
Also, few unused vars and imports were deleted in files where I made changes for translation.

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [X] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by wardi** [comment]
@mutantsan would you filter out the style changes and look in to the test failure?

Thanks!

**Comment by mutantsan** [comment]
@wardi sure, I'll do it soon

File: ckan/logic/action/create.py [hunk 1]
```diff
@@ -27,12 +25,10 @@
 import ckan.lib.datapreview
 import ckan.lib.api_token as api_token
 import ckan.authz as authz
-
-from ckan.common import _, config
-
-# FIXME this looks nasty and should be shared better
```
Comment (wardi) [inline ckan/logic/action/create.py:33]: I agree with the comment here about the import of `_update_package_relationship`, why are you removing it?

File: ckan/logic/action/delete.py [hunk 2]
```diff
@@ -310,12 +307,14 @@ def member_delete(context, data_dict=None):
...
-        raise NotFound('Group was not found.')
+        raise NotFound(_('Group was not found.'))
 
     obj_class = ckan.logic.model_name_to_class(model, obj_type)
     obj = obj_class.get(obj_id)
     if not obj:
-        raise NotFound('%s was not found.' % obj_type.title())
+        raise NotFound(_('{obj_type} was not found.').format(
+            obj_type=obj_type.title())
+        )
```
Comment (wardi) [inline ckan/logic/action/delete.py:317]: same comment as above

File: ckan/logic/action/update.py [hunk 3]
```diff
@@ -7,15 +7,14 @@
 import time
 import json
 
-from ckan.common import config
-import ckan.common as converters
 import six
 from six import text_type
 
-import ckan.lib.helpers as h
+import ckan.common as converters
```
Comment (wardi) [inline ckan/logic/action/update.py:13]: this looks wrong for sure

File: ckan/model/tag.py [hunk 4]
```diff
@@ -1,20 +1,21 @@
 # encoding: utf-8
 
 from sqlalchemy.orm import relation
-from sqlalchemy import types, Column, Table, ForeignKey, and_, UniqueConstraint
+from sqlalchemy import types, Column, Table, ForeignKey, UniqueConstraint
 
+import ckan  # this import is needed
```
Comment (wardi) [inline ckan/model/tag.py:6]: for...?

**Pull request closed without merging**